.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+        required: false
+        type: string
+        default: './.leviathan-inputs'
+      DEVICE_TYPE:
+        description: "slug of device type"
+        required: true
+        type: string
+      REPORTS:
+        description: "path to reports directory"
+        required: false
+        type: string
+        default: "./reports"
+      ENVIRONMENT:
+        description: "What BalenaCloud env, balena-cloud or balena-machine"
+        required: true
+        type: string
+        default: "balena-cloud"
+      BALENACLOUD_ORG:
+        required: false
+        description: "Organization used during cloud suite to create fleets, etc."
+        default: "testbot"
+        type: string
+      BALENACLOUD_APP_NAME:
+        description: "Testbot fleet for finding available Leviathan workers. Not used for QEMU workers. Can accept a list of apps separated by commas."
+        type: string
+        required: false
+        default: "balena/testbot-rig"
+      BALENACLOUD_API_URL:
+        type: string
+        required: false
+        default: "balena-cloud.com"
+      BALENAMACHINE_API_URL:
+        required: false
+        type: string
+        default: "bm.balena-dev.com"
+      BALENACLOUD_SSH_URL:
+        required: false
+        type: string
+        default: "ssh.devices.bm.balena-dev.com"
+      BALENACLOUD_SSH_PORT:
+        default: 222
+        type: number
+        required: false
+      QEMU_CPUS:
+        required: false
+        type: number
+        default: 4
+      QEMU_MEMORY:
+        type: string
+        required: false
+        default: "1G"
+      WORKER_TYPE:
+        description: "Valid choices: testbot/qemu"
+        type: string
+        required: true
+      LEVIATHAN_DEBUG:
+        description: |
+          Use this parameter to modify the default debugging settings on test runs. To debug test runs when rebuilding, add new values to this parameter separated with commas. Know more about the possible values available for debugging: https://balena-os.github.io/leviathan/pages/Getting-Started/debugging.html Example value: dev: true, failFast: false
+        type: string
+        required: false
+    secrets:
+      BALENACLOUD_API_KEY:
+        required: false
+      BALENAMACHINE_API_KEY:
+        required: false
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Get Leviathan checked out
+      uses: actions/checkout@v4
+      with:
+        repository: balena-os/leviathan
+        submodules: true
+        # path: ./leviathan-workspace
+        # The branch, tag or SHA to checkout. When checking out the repository that
+        # triggered a workflow, this defaults to the reference or SHA for that event.
+        # Otherwise, uses the default branch.
+        ref: ${{ inputs.LEVIATHAN_REF }}
+
+    # - name: Set up Docker Buildx
+    #   uses: docker/setup-buildx-action@v3
+    # Don't think, I need this since docker-compose uses only buildkit which we enabled by default in Makefile
```

Comment:
Install buildx anyway to be sure that the buildkit plugins are available.

Reply:
👍🏽